tests/internal/e2e/test/specs_env_exporter.ts

```diff
@@ -0,0 +1,17 @@
+import { getEnvVarsForGroup } from "./specs_config";
+
+const groupName = process.env.RUN_GROUP_NAME;
+let envVars: string;
+
+if (process.env.USE_VCAST_24 === "True") {
+  envVars = getEnvVarsForGroup(true, groupName);
+} else {
+  envVars = getEnvVarsForGroup(false, groupName);
+}
+
+if (envVars) {
+  console.log(envVars);
```

Comment:
Maybe we don't need this anymore, now that the things are working. Here you'd show all env vars. You might want to print only the ones that are set by the current test

Reply:
We need this because the runner fetches from the console log what he needs to export.